### PR TITLE
APIGOV-22850 - add new variable

### PR DESCRIPTION
**Pull request by jcollins-axway**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Added a new migration variable configuration option

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

### CODE REVIEW

File: content/en/docs/connect_manage_environ/connect_api_manager/agent-variables.md [hunk 1]
```diff
@@ -147,6 +147,7 @@ You can extend the previous minimum variable list with the following variables.
...
 | CENTRAL_SUBSCRIPTIONS_NOTIFICATIONS_SMTP_UNSUBSCRIBEFAILED_BODY    | The body of the email for Failed to Unsubscribe events.                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| CENTRAL_MIGRATION_CLEANINSTANCES                                   | When set to true the agent, on startup, will clean all but the latest API Service Instance for every API Service and Stage combination                                                                                                                                                                                                                                                                                                                                                                      |
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/connect_api_manager/agent-variables.md:150]: instance(s)

File: content/en/docs/connect_manage_environ/connect_aws_gateway/deploy-your-agents-1.md [hunk 2]
```diff
@@ -115,6 +115,7 @@ The configuration of the AWS_QUEUENAME is not used in this mode.
...
 | CENTRAL_SUBSCRIPTIONS_NOTIFICATIONS_SMTP_UNSUBSCRIBEFAILED_BODY    | The body of the email for Failed to Unsubscribe events, see [SMTP notifications](#smtp-notifications).                                                                                                                                                                                                                                                                                                                                                                                             |
+| CENTRAL_MIGRATION_CLEANINSTANCES                                   | When set to true the agent, on startup, will clean all but the latest API Service Instance for every API Service and Stage combination                                                                                                                                                                                                                                                                                                                                                             |
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/connect_aws_gateway/deploy-your-agents-1.md:118]: Instance(s)

**Review by robertengaxway — APPROVED** [review]
made minor suggestion around "instance(s)"

File: content/en/docs/connect_manage_environ/connect_api_manager/agent-variables.md (hunk 1 above)
Comment (jcollins-axway) [reply content/en/docs/connect_manage_environ/connect_api_manager/agent-variables.md:150]: This should be a single instance, maybe changing 'but' to 'except' will clarify it more?

@lbadenhop would this be better? "...clean all except the latest API Service Instance..."

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**